cmd/piper.go

```diff
@@ -203,6 +203,7 @@ func addRootFlags(rootCmd *cobra.Command) {
 	rootCmd.PersistentFlags().StringVar(&GeneralConfig.VaultServerURL, "vaultServerUrl", "", "The vault server which should be used to fetch credentials")
 	rootCmd.PersistentFlags().StringVar(&GeneralConfig.VaultNamespace, "vaultNamespace", "", "The vault namespace which should be used to fetch credentials")
 	rootCmd.PersistentFlags().StringVar(&GeneralConfig.VaultPath, "vaultPath", "", "The path which should be used to fetch credentials")
+	rootCmd.PersistentFlags().StringVar(&GeneralConfig.StepMetadata, "stepMetadata", "", "Step metadata, passed as path to yaml")
```

Comment:
Why do we have this parameter now? Does one need to pass is now if I call piper?

Reply:
It was added just like getConfig.go (L193), but we can discuss it

Reply:
I guess we could get rid of this parameter. Retrieving step metadata is sufficient via the name of the step.